app/src/common/shared/org/mozilla/vrbrowser/ui/views/AutoCompletionView.java

```diff
@@ -60,6 +62,8 @@ public AutoCompletionView(Context aContext, AttributeSet aAttrs, int aDefStyle)
     private void initialize(Context aContext) {
         inflate(aContext, R.layout.autocompletion_bar, this);
         mFirstLine = findViewById(R.id.autoCompletionFirstLine);
+        mLineWidth = WidgetPlacement.dpDimension(getContext(), R.dimen.keyboard_width);
+        mPerKeyWidth = WidgetPlacement.dpDimension(getContext(), R.dimen.keyboard_key_width);
```

Comment:
I am trying to get the correct line width from `mFirstLine `, but it always returns zero from `getWidth()` or `getmesakuredwidth()`. `R.dimen.keyboard_width ` is not exactly correct because our real width should be the same with `world_width`.

Reply:
@MortimerGoro I am trying to use post runnable to make sure this UI has been loaded. but it still shows zero.
```
       mFirstLine.post(() -> {
            mLineWidth = mFirstLine.getWidth(); // the width is still not ready...
        });
```
Alternatively, `R.dimen.world_width` is more close to the correct result.